fluentui_drawer/src/main/java/com/microsoft/fluentui/bottomsheet/BottomSheetItem.kt

```diff
@@ -55,7 +57,8 @@ class BottomSheetItem : Parcelable {
         customBitmap: Bitmap? = null,
         disabled: Boolean = false,
         @DrawableRes accessoryImageId: Int = NO_ID,
-        accessoryBitmap: Bitmap? = null
+        accessoryBitmap: Bitmap? = null,
+        roleDescription: String = "Button",
```

Comment:
what should be the default value? If user does not use this parameter, wht should accessibility announce

Reply:
empty string has been made default